src/pages/SignUp/SignUp.jsx

```diff
@@ -1,13 +1,153 @@
+import "./SignUp.css"
 import { Navbar } from "../../components/Navbar/Navbar";
+import { useState } from "react";
+import { signupService } from "../../services/signupService";
+import { useAuth } from "../../context/authContext";
+import { Link } from "react-router-dom";
 
-const Signup = () => {
+const SignUp = () => {
+  const [signupInput, setSignupInput] = useState({});
+  const { setIsAuth, setToken, navigate } = useAuth();
+  const [hide, setHide] = useState({ pswd: true, confirmpswd: true });
```

Comment:
You can use  camelCasing for `confirmpswd`

Reply:
Ok i will do that, thankyou for pointing it out